Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= oamdev/terraform-controller:0.2.8
+IMG ?= oamdev/terraform-controller:0.2.14
```

Comment:
Please keep this line as it is. I am writing a script to automatically replacing 0.2.8 to its latest tag.

Reply:
updated